renovate.json5

```diff
@@ -35,7 +37,15 @@
                 'System.Text.Json',
             ],
             allowedVersions: '<9.0.0',
-            matchCurrentVersion: '<9.0.0',
+            matchCurrentVersion: '/^8\\./',
```

Comment:
Note that `System.IO.Hashing` and `System.Runtime.Caching` are also on 8.x and will be upgraded to 9.x. These two aren't in the target framework-specific blocks, so this is appropriate, but I'm wondering if this is what we actually want, or if we want to do the same v8-v9 distinction with them.

Reply:
I am not really sure. But it may be best to list the few packages manually instead of using regular expression here so we know the exact packages that we purposely don't want to upgrade

Reply:
Those are listed, check line 33 and below. The change here is only about fixing the package rule so it actually works and prevents those packages from being updated from 8.x to 9.x.

Reply:
Line 34 and 35 are regular expression which could cause unwanted behavior.

I am not familiar with renovate, so I trust you

Reply:
I think we need the v8-v9 distinction at least for `System.IO.Hashing`. I've been running into problems e.g. with Aspire that is using the newer version 9.x.

Reply:
Hmm, so we should update `System.IO.Hashing` to v9 for .NET 9? Sounds reasonable, though I wonder why we didn't have it like that already (barring nobody thinking of it).

`System.Runtime.Caching` I've actually removed, because `Microsoft.Data.SqlClient` doesn't use it anymore. I removed a few other similar unnecessary references too.

Reply:
Smoke tested some features using `System.IO.Hashing`, they still work.